pkgs/by-name/or/or-tools/pybind11-2.13.6.nix

```diff
@@ -0,0 +1,131 @@
+# This is pybind11 2.13.6 from an earlier nixpkgs version, with the
+# parts from https://github.com/google/or-tools/commit/7f29b27840436e19b6530d5c7f23eeadd819bd3e
+# applied.
+{
+  stdenv,
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+  setuptools,
+  boost,
+  eigen,
+  python,
+  catch2,
+  numpy,
+  pytestCheckHook,
+  libxcrypt,
+  makeSetupHook,
+}:
+let
+  setupHook = makeSetupHook {
+    name = "pybind11-setup-hook";
+    substitutions = {
+      out = placeholder "out";
```

Comment:
Unused, I think

Reply:
This is simply an old version from nixpkgs without any changes. We can do another round of cleanups in a followup.